MaeBlock.hpp

```diff
@@ -234,7 +235,7 @@ class EXPORT_MAEPARSER Block
         return get_property<std::string>(m_smap, name);
     }
 
-    void setStringProperty(const std::string& name, std::string value)
+    void setStringProperty(const std::string& name, const std::string& value)
```

Comment:
I thought the idiom on the lhs was preferred (with a std::move in the body). The rhs always incurs a copy, the lhs may not.

Reply:
Good catch!